src/show/components/Navbar/index.js

```diff
@@ -1,33 +1,69 @@
-import React from "react";
+import { React, useState } from "react";
 import { Link, useNavigate } from "react-router-dom";
-
+import { XMarkIcon } from '@heroicons/react/24/solid';
 import { Cart } from "../../assets/svg";
+import MenImage from "../../assets/img/men.png";
+import WomenImage from "../../assets/img/women.jpg";
+import KidsImage from "../../assets/img/kid.png";
```

Comment:
Can't we do the same in case of image that we are doing for the svgs. Import all image at index and then export. then here 
`import {Men, Women, Kids} from "../../assets/img"`

Reply:
Sorry sir, I remembered this, but I didn't know that I could import different images like this.